src/test/scala/edu/ie3/simona/model/grid/RefSystemSpec.scala

```diff
@@ -6,43 +6,56 @@
 
 package edu.ie3.simona.model.grid
 
-import org.scalatest.flatspec.AnyFlatSpec
+import edu.ie3.simona.test.common.UnitSpec
 import org.scalatest.matchers.should.Matchers
-import squants.{Dimensionless, Each}
-import squants.electro.{Amperes, ElectricPotential, Kilovolts, Ohms}
-import squants.energy.{Megawatts, Power, Kilowatts}
+import squants.{Dimensionless, Each, ElectricCurrent}
+import squants.electro.{
+  Amperes,
+  ElectricPotential,
+  ElectricalResistance,
+  Kilovolts,
+  Ohms,
+}
+import squants.energy.{Kilowatts, Megawatts, Power}
+
+class RefSystemSpec extends UnitSpec with Matchers {
+
+  implicit val tolerance: Dimensionless = Each(1e-12)
+  implicit val currentTolerance: ElectricCurrent = Amperes(1e-9)
+  implicit val impedanceTolerance: ElectricalResistance = Ohms(1e-9)
 
-class RefSystemSpec extends AnyFlatSpec with Matchers {
+  "A RefSystem with nominal power and nominal voltage" should {
 
-  "A RefSystem with nominal power and nominal voltage" should "provide corresponding nominal current and nominal impedance" in {
+    "provide corresponding nominal current and nominal impedance" in {
 
-    val nominalPower: Power = Kilowatts(600)
-    val nominalVoltage: ElectricPotential = Kilovolts(10)
+      val nominalPower: Power = Kilowatts(600)
+      val nominalVoltage: ElectricPotential = Kilovolts(10)
 
-    val refSystem = RefSystem(nominalPower, nominalVoltage)
+      val refSystem = RefSystem(nominalPower, nominalVoltage)
 
-    refSystem.nominalPower should be(nominalPower)
-    refSystem.nominalVoltage should be(nominalVoltage)
-    refSystem.nominalCurrent should be(
-      Amperes(34.64101615137754774109785366023500d)
-    )
-    refSystem.nominalImpedance should be(
-      Ohms(166.6666666666666666666666666666666d)
-    )
+      refSystem.nominalPower should be(nominalPower)
+      refSystem.nominalVoltage should be(nominalVoltage)
+      refSystem.nominalCurrent should approximate(
+        Amperes(34.64101615137755)
+      )
+      refSystem.nominalImpedance should approximate(
+        Ohms(166.66666666666666)
+      )
+    }
   }
 
-  "A dimensionless impedance" should "be transferred correctly between reference systems" in {
+  "A dimensionless impedance be transferred correctly between reference systems" in {
```

Comment:
The test name now does not make much sense. Please rephrase it so it with `A RefSystem ... should ...`